### ISSUE TITLE
Trying to filter a column with too many disparate and long values can hang the UI.

### ISSUE THREAD

**Issue by alexbfree** [hestiaAI/hestialabs-experiences#935]
In my case, I was on the Instagram experience, the Timeline tab, where 9251 rows are in the table. I clicked Filter icon on Description column, and the app hung. Presumably as it tries to generate a ridiculous large filter popup on values that will be all different anyway!

It did eventually recover (after i clicked Wait for page on the resulting crash popup and waited ~ 2 mins)

**Comment by valentinoli** [comment]
We need have a way to configure which columns are filterable

**Comment by valentinoli** [comment]
Leaving in "Assigned" for now with a low priority

**Comment by valentinoli** [comment]
Leaving in "Development needed" for now

**Comment by alexbfree** [comment]
I propose that we move #46, #47 and #48 (three small low priority tickets) to either Specs Done or Committed to Dev, so that we stop discussing them everyday.

Logic for moving to specs done would be that the ticket is actionable by dev but no decision to proceed has been taken yet. Downside is that decision is left with bizdev

Logic for moving to committed to dev is that it is most definitely on dev's plate and requires no further discussion. Downside is that it builds up a large pile of work in Committed (but this could also be seen as an advantage, in that there will always be work that's definitely assigned to dev that can be picked up)

On balance I think moving to 'specs done' would be better - provided Bizdev are ok with us leaving it there and are ok with us occasionally grabbing such uncontroversial low priority items from 'their' column.

**Comment by alexbfree** [comment]
Issue observed in Instagram Experience.

- Timeline Tab
  - Description Column
  - File name Column

UI also hangs when switching to a different tab (and indeed all subsequent tab switches are affected too) from the Timeline tab, when this many rows are loaded (which may or may not need a separate issue).

**Comment by alexbfree** [comment]
Per @valentinoli suggestion: let's convert this to a specified requirement, where we can specify for each column whether it is filterable or not, as part of the experience config. We can then set these columns to be non-filterable.

The hanging when you switch between tabs can be looked at separately once this has been addressed. should be done in parallel to #997.

**Comment by alexbfree** [comment]
Also experienced hanging when moving between tabs on a large file within Google experience. (filtering columns was OK on that experience tho)
